config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml

```diff
@@ -10,7 +10,7 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-kubeadm,sig-release-master-informing
     testgrid-tab-name: kubeadm-kinder-master-on-1-18
-    testgrid-alert-email: kubernetes-sig-cluster-lifecycle+testgrid@googlegroups.com
+    testgrid-alert-email: kubernetes-sig-cluster-lifecycle+testgrid@googlegroups.com, kubernetes-release-team@googlegroups.com
```

Comment:
if the email is added to all the jobs in kubeadm-kinder-upgrade.yaml shouldn't the same apply to these files?
kubeadm-kinder-x-on-y.yaml 
kubeadm-kinder.yaml

Reply:
Definitely. Thanks for catching that :+1: I have pushed update